recipes/libssh/all/conanfile.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import collect_libs, copy, get, rmdir
+
+
+class LibSSHRecipe(ConanFile):
+    name = "libssh"
+    version = "0.10.4"
+    license = "LGPL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.libssh.org/"
+    description = "multiplatform C library implementing the SSHv2 protocol on client and server side"
+    topics = ("libssh", "ssh", "shell", "ssh2", "connection")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_zlib": [True, False],
+        "crypto_backend": ["openssl", "gcrypt"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_zlib": True,
+        "crypto_backend": "openssl",
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
```

Comment:
```suggestion

    def configures(self):
        if self.options.shared:
            del self.options.fPIC

```
Shared library always should use fPIC, so it does affect the package ID.